### ISSUE TITLE
wuauclt.exe not present in Windows Server 2016 core install

### ISSUE THREAD

**Issue by rhymeswithmogul** [MicrosoftDocs/windowsserverdocs#4838]
I was able to follow the Server Core updating guide all the way to the final step, where it told me that `Wuauclt` could not be found.  This is a default (core) install of Windows Server 2016 Standard.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 35b743aa-a61e-2ecb-ef34-2d6fd31c836b
* Version Independent ID: 6dcc4daa-056f-3d9a-8e7a-ba2f584a815b
* Content: [Patching Server Core](https://docs.microsoft.com/en-us/windows-server/administration/server-core/server-core-servicing)
* Content Source: [WindowsServerDocs/administration/server-core/server-core-servicing.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/administration/server-core/server-core-servicing.md)
* Product: **windows-server**
* Technology: **administration**
* GitHub Login: @pronichkin
* Microsoft Alias: **artemp**

**Comment by illfated** [comment]
Hmm. I wonder if we should update the article to use the PowerShell command to force Windows Update to run instead, because as far as I know, `wuauclt.exe` is referred to as deprecated several places and some support sites also lists it as not working anymore (more likely to fail than to succeed when used).

I tried searching for how to manually install `wuauclt.exe`, but I kept getting too many search results about how to remove Windows viruses and trojans disguised as `wuauclt.exe`. That is when I noticed that it might be easier to search for how to trigger the Settings components of Windows Update via PowerShell instead of using `wuauclt.exe`.

**Comment by exchange12rocks** [comment]
Shouldn't we just use `usoclient StartScan`?

**Comment by illfated** [comment]
> Shouldn't we just use `usoclient StartScan`?

Looks reasonable to me. Could we reference some other article where this is a recommended usage, if we start a PR to have that command replace the `wuauclt.exe` methods?

**Comment by exchange12rocks** [comment]
@illfated I am not sure that's an official way. Maybe MS doesn't actually want us to use that command

**Comment by illfated** [comment]
Fair enough. I didn't know that, so it is a good thing that you mentioned that aspect. Not sure what we can do before we hear from someone in the MS Docs team.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
